#! /bin/sh /usr/share/dpatch/dpatch-run
## ubuntu_nofuse_fix_photo_import.dpatch by Sebastien Bacher <seb128@ubuntu.com>
##
## DP: Set X-GIO-NoFuse in the import desktop so gvfs uri are correctly used
## DP: LP: #338466

@DPATCH@
diff -urNad f-spot-0.5.0.3~/f-spot-import.desktop.in f-spot-0.5.0.3/f-spot-import.desktop.in
--- f-spot-0.5.0.3~/f-spot-import.desktop.in	2008-10-18 13:48:28.000000000 +0200
+++ f-spot-0.5.0.3/f-spot-import.desktop.in	2009-03-10 15:39:39.000000000 +0100
@@ -17,3 +17,4 @@
 X-GNOME-Bugzilla-Version=0.5.0.3
 X-GNOME-DocPath=f-spot/f-spot.xml
 X-SuSE-translate=true
+X-GIO-NoFuse=true
diff -urNad f-spot-0.5.0.3~/f-spot-import.desktop.in.in f-spot-0.5.0.3/f-spot-import.desktop.in.in
--- f-spot-0.5.0.3~/f-spot-import.desktop.in.in	2008-09-26 10:27:39.000000000 +0200
+++ f-spot-0.5.0.3/f-spot-import.desktop.in.in	2009-03-10 15:39:43.000000000 +0100
@@ -17,3 +17,4 @@
 X-GNOME-Bugzilla-Version=@VERSION@
 X-GNOME-DocPath=f-spot/f-spot.xml
 X-SuSE-translate=true
+X-GIO-NoFuse=true
